build/buildPlatformBinaries.sh

```diff
@@ -23,6 +23,9 @@ case $PLATFORM_TO_BUILD in
 		;;
 	'nodejs')
 		"$platformsDir/nodejs/getNode.sh"
-		;;            
+		;; 
+	'ruby')
```

Comment:
Make sure to create a container for 'ruby' before running the platform-binary-build for it. BTW, create a new pipeline for ruby in `vsts/platformbinaries` folder and I can create the build definition for it.